src/__tests__/three-idx.test.ts

```diff
@@ -0,0 +1,312 @@
+import tmp, { DirectoryResult } from 'tmp-promise'
+import Ceramic from '@ceramicnetwork/core'
+import { DID } from 'dids'
+import { Ed25519Provider } from 'key-did-provider-ed25519'
+import KeyResolver from 'key-did-resolver'
+import Ipfs from 'ipfs'
+import all from 'it-all'
+import { schemas, definitions } from '@ceramicstudio/idx-constants'
+import { publishIDXConfig } from '@ceramicstudio/idx-tools'
+import { randomBytes } from '@stablelib/random'
+import ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
+
+import { NewAuthEntry, ThreeIDX } from '../three-idx'
```

Comment:
Same thing here. Is there a way to make the diff not disappear?

Reply:
Uhh, let me prepare then a PR with a different order of operations.

Reply:
Thanks 🙏